packages/mobile/src/fiat-connect/types.ts

```diff
@@ -0,0 +1,94 @@
+export enum FiatType {
+  USD = 'USD',
+  EUR = 'EUR',
+}
+
+export enum CryptoType {
+  cUSD = 'cUSD',
+  cEUR = 'cEUR',
+  CELO = 'CELO',
+}
+
+export enum KycSchema {
+  NameAndAddress = 'NameAndAddress', // NOTE: this is a MOCK schema (not a real one in the spec!)
+}
+
+export interface NameAndAddressKYCSchema {
+  // NOTE: this is a MOCK schema (not a real one in the spec!)
+  firstName: string
+  lastName: string
+  address: PostalAddress
+}
+
+export interface PostalAddress {
+  address1: string
+  address2?: string
+  city: string
+  region: string // in the US this means state
+  postalCode: string
+  isoCountryCode: string
+}
+
+export enum FeeType {
+  KYCFee = 'KYCFee',
+  PlatformFee = 'PlatformFee',
+}
+
+export enum FeeFrequency {
+  OneTime = 'OneTime',
+  Recurring = 'Recurring',
+}
+
+export enum FiatAccountType {
+  CheckingAccount = 'CheckingAccount',
+  DebitCard = 'DebitCard',
+  CreditCard = 'CreditCard',
+}
+
+export enum FiatAccountSchema {
+  CheckingAccount = 'CheckingAccount', // NOTE: this is a MOCK schema (not a real one in the spec!)
+}
+
+export interface CheckingAccountSchema {
+  // NOTE: this is a MOCK schema (not a real one in the spec!)
+  bankName: string
+  accountName: string
+  fiatType: FiatType
+  accountNumber: string
+  routingNumber: string
+}
+
+export enum TransferType {
+  TransferIn = 'TransferIn',
+  TransferOut = 'TransferOut',
+}
+
+export enum TransferStatus {
+  TransferStarted = 'TransferStarted',
+  TransferPending = 'TransferPending',
+  TransferComplete = 'TransferComplete',
+  TransferFailed = 'TransferFailed',
+}
+
+export enum WebhookEventType {
+  // NOTE: if we use these we should update the spec (which has 'webhook' prefix for all of these)
+  KYCStatusEvent = 'KYCStatusEvent',
+  TransferInStatusEvent = 'TransferInStatusEvent',
+  TransferOutStatusEvent = 'TransferOutStatusEvent',
+}
+
```

Comment:
When I thought about this ticket, I thought it would also include all of the Request Body, error objects, and response objects for the different endpoints such as `QuoteRequest`, `QuoteResponse`, `QuoteErrorResponse`, `TransferRequest`, `TransferResponse`, etc.

What do others think? @cajubelt @lisaqyli @jophish 

A Pro is that it is very useful to have those explicitly typed so that client libraries and partners can implement the endpoints/functions with confidence. A con is that its a bit of extra work to maintain while the endpoints are still in flux.

Reply:
Hmm... this is a good point. The SDK will need to expose `xxxRequest` types to allow clients to pass in arguments to API methods, and `xxxResponse` types to return values... the API will also need exactly these types in order to implement its endpoints. I would say +1 on including those types here.